earth_enterprise/src/scons/getversion.py

```diff
@@ -222,17 +244,36 @@ def set_long(self, value):
 
         self.long_version_string = value
 
+    def is_warning_available(self):
+        """Returns whether a warning message should be shown."""
+
+        return not _IsFirstParentGitDescribeSupported()
+
+    def get_warning_message(self):
+        """Returns any issues with version calculation with mitigation steps."""
```

Comment:
Minor: This docstring should specify the value returned when there are no warnings.  `None` is an alternative for indicating no warnings.